tests/conftest.py

```diff
@@ -1,7 +1,17 @@
+import pathlib
+
 import pytest
 import os
 
+import finalfusion.vocab
+
 
 @pytest.fixture
 def tests_root():
-    yield os.path.dirname(__file__)
+    yield pathlib.PurePath(os.path.dirname(__file__))
+
+
+@pytest.fixture
+def simple_vocab_fifu(tests_root):
+    yield finalfusion.vocab.load_vocab(
+        os.path.join(tests_root, "data/simple_vocab.fifu"))
```

Comment:
A forward slash should work here too.

Reply:
Or maybe it requires changing the common test code. If so, then it should probably go into a separate PR.

Reply:
I actually snuck in the common test code change by wrapping `os.path.dirname(__file__)` in `PurePath`, just added the "/" here as well.